CONTRIBUTING.md

```diff
@@ -62,28 +62,34 @@ discuss pending contributions in order to find a resolution. It is expected that
 small minority of issues be brought to the WG for resolution and that discussion and
 compromise among collaborators be the default resolution mechanism.
 
-# Becoming a Collaborator
+# Becoming a Member and Collaborator
 
-All contributors who land a non-trivial contribution should be on-boarded in a timely manner,
+To become a member of the Community Committee, start participating! Attend the bi-weekly meetings, 
+file issues and pull requests, and provide insight via GitHub. Request to become an Observer by 
```

Comment:
What I remember from a convo we had in Berlin was that we were looking to "not over think it" here. What I *think* that means is - anyone can request it by being bold enough to opening an issue (which would include discussion with them- and maybe conclude that they don't want to be on the committee). If some complete stranger wants to open an issue requesting it- they can. Beyond this we leave it as a "good faith" process.

Yes, maybe we'll see someone that just attends 1 meeting- but I think it's not that big of an overhead burden. We may want to add something like "1 request per year" for good measure though.

Also- I think the chairperson should just silently prune observers that don't make minimum requirements.